app/styles/ui/_list.scss

```diff
@@ -16,6 +16,18 @@
   .ReactVirtualized__Grid {
     background: var(--background-color);
   }
+
+  .keyboard-insertion-element {
+    display: block;
+    position: fixed;
+    z-index: 1000;
+  }
+
+  .overlay {
+    position: fixed;
+    z-index: 1000;
```

Comment:
😭 Could we instead add an entry for this z-index here: https://github.com/desktop/desktop/blob/92aa93548109404d0855318bb2c355dd06b41af7/app/styles/_variables.scss#L334-L348 so that we can keep track of what actually needs to go on top (probably tooltips should still be on top of this for example)

Reply:
Yes! Sorry, that `1000` was a temporary thing, I intended to either reuse or come up with a new z-index constant where you suggest but forgot about it 😳  I think I will reuse `--drag-overlay-z-index`

Reply:
Fixed in [3e44157](https://github.com/desktop/desktop/pull/17671/commits/3e441579f36de53d3d83e06e2a66f9b7ff0f5bea)!